src/cil.ml

```diff
@@ -1444,26 +1414,26 @@ type attributeClass =
  * conversion *)
 let attributeHash: (string, attributeClass) H.t =
   let table = H.create 13 in
-  List.iter (fun a -> H.add table a (AttrName false))
+  List.iter (fun a -> H.add table a AttrName)
     [ "section"; "constructor"; "destructor"; "unused"; "used"; "weak";
       "no_instrument_function"; "alias"; "no_check_memory_usage";
       "exception"; "model"; (* "restrict"; *)
       "aconst"; "__asm__" (* Gcc uses this to specify the name to be used in
                            * assembly for a global  *)];
 
   (* Now come the MSVC declspec attributes *)
-  List.iter (fun a -> H.add table a (AttrName true))
+  List.iter (fun a -> H.add table a AttrName)
     [ "thread"; "naked"; "dllimport"; "dllexport";
       "selectany"; "allocate"; "nothrow"; "novtable"; "property";  "noreturn";
       "uuid"; "align" ];
```

Comment:
These were previously separated by `separateStorageModifiers` in front, as they were mapped to `AttrName true`, right?

Reply:
Yes, but it seems they need not be separated on GCC.

Reply:
Are all of these MSVC-specific? C11 introduced also a `noreturn` / `_Noreturn`, but this is probably not on the base branch?

Reply:
If they are MSVC specific, one should remove them probably.

Reply:
I removed the ones that are MSVC specific now. GCC picked up support for a lot of them over the years for compatibility purposes. 

`noreturn` is in fact supported via this, it has also been a GCC extension for a long time. In the C11 branch, we reuse this pre-C11 support to handle it.

Reply:
Good! I think then the comment above: `   (* Now come the MSVC declspec attributes *)` is obsolete or should be changed to indicate that this is still needed for GCC?

Reply:
Good! I think then the comment above: ` (* Now come the MSVC declspec attributes *)` is obsolete or should be changed to indicate that this is still needed for GCC?